engines/mysql.py

```diff
@@ -91,6 +93,25 @@ def table_exists(self, dbname, tablename):
                 self.existing_table_names.add((schema.lower(), table.lower()))
         return (dbname.lower(), tablename.lower()) in self.existing_table_names
 
+    def to_csv(self):
+        csvfile_output = os.path.normpath(self.table_name() + '.csv')
+        self.get_cursor()
+        sql_query = ("SELECT * FROM " + self.table_name() + ";")
+        self.cursor.execute(sql_query)
+        row = self.cursor.fetchone()
+        colnames = [tuple_i[0] for tuple_i in self.cursor.description]
+        csv_out = open(csvfile_output, "wb")
+        csv_writer = csv.writer(csv_out, dialect='excel')
+        csv_writer.writerow(colnames)
+
+        while row is not None:
+            csv_writer.writerow([values for values in row])
```

Comment:
What is `row`'s type? I'm guessing it's a tuple in which case `[values for values in row]` is just converting it to a list? It seems like `.writerow(row)` ought to work, or if not `.writerow(list(row))`, but I could definitely be missing something.